includes/discovery/sensors/entity-sensor.inc.php

```diff
@@ -185,6 +185,11 @@
             if (isset($entry['entPhySensorOperStatus']) && ($entry['entPhySensorOperStatus'] === 'unavailable' || $entry['entPhySensorOperStatus'] === 'nonoperational')) {
                 $valid_sensor = false;
             }
+
+            if ($entry['entPhySensorValue'] == '-1000000000') {
```

Comment:
Shouldn't we check the OS we are in, before assuming this value makes this sensor invalid ? Adding a meaning to a value is very device specific, and I am pretty sure a device exists with this value being legitimate.

Reply:
@PipoCanaja When I first wrote the code it was gated behind an OS, but then I read a few MIB definitions and decided this should be appropriate.